src/actions/game_actions.js

```diff
@@ -15,26 +12,31 @@ export const SUBMIT_TRIAL = 'SUBMIT_TRIAL';
 export const START_LEVEL = 'START_LEVEL';
 export const RECEIVE_PLAYED_LEVELS_INFO = 'RECEIVE_PLAYED_LEVELS_INFO';
 
+var levelsConfigurationFile = require('../../assets/levels.json');
+
+function obtainLevels() {
+    let levels = {};
+    for (let [levelNumber, probabilityPerCategories] of Object.entries(levelsConfigurationFile)) {
```

Comment:
@MaxiSuppes En ES6 hay iteradores mas copados onda el "each" para no tener que hacer "for"s. Ya los fors son muy anticuados y molestos de leer, si podés cambialos.

Reply:
@lucianolev Perfecto